index.html

```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 <html>
   <head>
-    <base href="/">
+    <base href=".">
```

Comment:
I tried this change on a project and it broke loading a page from the direct url (example.com/url/to/path)